app/services/cart.js

```diff
@@ -151,6 +151,7 @@ export default ApiService.extend(asyncMixin, {
    */
   refresh() {
     this.get("store").unloadAll("requested_package");
+    this.get("store").findAll("package", { reload: true });
```

Comment:
What is the reason to fetch `package` here ?

Reply:
To get the remaining quantities of the packages after the order is placed